### PR TITLE
add localization files and pass loc arg through

### PR DESCRIPTION
**Pull request by rihorn2**

### CODE REVIEW

File: fairlearn/widget/_fairlearn_dashboard.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ def __init__(
             self, *,
             sensitive_features,
             y_true, y_pred,
-            sensitive_feature_names=None):
+            sensitive_feature_names=None,
+            locale=None):
```
Comment (romanlutz) [inline fairlearn/widget/_fairlearn_dashboard.py:58]: Do we want to add a line to CHANGES.md for this?

File: visualization/FairnessDashboard/src/Localization/zh-TW.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    "precisionDescription": "在分類為 1 的資料點當中，正確分類的資料點。",
+    "recallDescription": "在 True 標籤為 1 的資料點當中，正確分類的資料點。替代名稱: 確判為真率、敏感度。",
+    "rmseDescription": "平方誤差平均的平方根。",
+    "mseDescription": "平方誤差的平均。",
+    "meanAbsoluteErrorDescription": "誤差絕對值的平均。比 MSE 更不易受極端值影響。",
+    "r2Description": "模型所解釋的標籤變異數。",
+    "aucDescription": "從負值範例分隔出正值範例的預測品質，以分數檢視。",
+    "balancedRMSEDescription": "正值與負值範例會重新加權，使總權重相等。適用於高度不平衡的基礎資料。",
+    "balancedAccuracyDescription": "正值與負值範例會重新加權，使總權重相等。適用於高度不平衡的基礎資料。",
+    "f1ScoreDescription": "F1-Score is the harmonic mean of precision and recall."
```
Comment (romanlutz) [inline visualization/FairnessDashboard/src/Localization/zh-TW.json:144]: Doesn't this need a translation?

File: visualization/FairnessDashboard/src/Localization/zh-TW.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    "globalAccuracyText": "為整體 {0}",
+    "accuracyDisparityText": "為 {0} 中的差距",
+    "editConfiguration": "編輯組態",
+    "backToComparisons": "多重模型檢視",
+    "outcomesTitle": "預測中的差距",
+    "minTag": "最小值",
+    "maxTag": "最大值",
+    "groupLabel": "子群組",
+    "underestimationError": "低於預測",
+    "underpredictionExplanation": "(已預測 = 0，True = 1)",
```
Comment (romanlutz) [inline visualization/FairnessDashboard/src/Localization/zh-TW.json:81]: False gets a translation, and True doesn't?

File: visualization/FairnessDashboard/src/Localization/zh-TW.json (hunk 2 above)
Comment (romanlutz) [reply visualization/FairnessDashboard/src/Localization/zh-TW.json:144]: actually this is in every language

File: visualization/FairnessDashboard/src/Localization/zh-CN.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  "calculating": "正在计算...",
+  "accuracyMetric": "性能指标",
+  "errorOnInputs": "输入出错。目前敏感特征必须是分类值。请将值映射到已装箱的类别，然后重试。",
+  "Accuracy": {
+    "header": "你想如何衡量性能?",
+    "modelMakes": "模型",
+    "modelsMake": "模型",
+    "body": "你的数据包含 {0} 个标签和 {2} {1} 预测。根据此信息，我们推荐以下指标。请从列表中选择一个指标。",
+    "binaryClassifier": "二元分类器",
+    "probabalisticRegressor": "Probit 回归量",
```
Comment (romanlutz) [inline visualization/FairnessDashboard/src/Localization/zh-CN.json:26]: nit: probabilistic - but if it's hard to fix the name then leave it for a future PR

File: visualization/FairnessDashboard/src/Localization/zh-TW.json (hunk 3 above)
Comment (rihorn2) [reply visualization/FairnessDashboard/src/Localization/zh-TW.json:81]: I thought that was particularly odd...

File: visualization/FairnessDashboard/src/Localization/zh-TW.json (hunk 2 above)
Comment (rihorn2) [reply visualization/FairnessDashboard/src/Localization/zh-TW.json:144]: yes, I think they left all technical phrases untranslated for the time.

**Pull request merged**